### DISCUSSION TITLE
Domain issue support

### DISCUSSION THREAD

**Discussion by enmartinson** [vercel/community#248] [Help] [answered]
Not sure what to do about this... put in a ticket for an obvious bug and I can't update it, respond to it or escalate it... just have to wait until someone decides to act on it within the next 2 days. In the meantime I have a production site ready to launch and a customer expecting it to, but have this stupid error that won't clear:
![domain-error](https://user-images.githubusercontent.com/486563/132602910-c67f9216-5eae-415e-9714-2e53553497e8.jpg)
It won't let me remove and re-add the domain, I've tried a dozen times. A popup tells me the delete is successful but never is. Adding the domain back in fails. Anyone with support access it's ticket #00044324, it sure would be nice to have this bug cleared out so I can actually deploy the site.

**Comment by leerob** [comment, marked as answer]
Hey, sorry for the slow response here, but I see that we did get back to you on the support ticket 👍

**Reply by Pieparker** [reply]
This topic is rather old and has been marked as Answered. If anybody else is exhibiting the same or a similar issue to the original poster, please feel free to create your own discussion and provide as much context as you can. This helps us to ensure that discussions are relevant, fresh and free of any possibly misleading context.
